package.json

```diff
@@ -14,6 +14,8 @@
     "@react-native-community/clipboard": "^1.5.1",
     "@react-native-community/datetimepicker": "^3.0.8",
     "@react-native-community/masked-view": "^0.1.10",
+    "@react-native-community/progress-bar-android": "^1.0.4",
+    "@react-native-community/progress-view": "JaneaSystems/progress-view#windows-fix",
```

Comment:
Are these dependencies needed for this PR?

Reply:
Yes, not having progress view causes issues.